Sources/VaporJWT/Droplet+JWT.swift

```diff
@@ -0,0 +1,27 @@
+import Vapor
+import JWT
+
+extension Droplet {
+    /// Returns the main JWT signer
+    /// or throws an error if not properly configured
+    public func jwtSigner() throws -> Signer {
+        guard let signer = storage["jwtSigner"] as? Signer else {
+            throw ConfigError.noSigner()
+        }
+
+        return signer
+    }
+
+    /// Verifies the signature of a JWT
+    /// using the droplet's main JWT signer
+    public func verifySignature(jwt: JWT) throws {
+        let signer = try jwtSigner()
+        try jwt.verify(using: signer)
+    }
+
+    /// Used internally to set the droplet's
+    /// main JWT signer
+    internal func setJWTSigner(signer: Signer) {
```

Comment:
there might be some integrations I'm not aware of, but any reason for this to not follow the variable pattern we use elsewhere? Is it a way of forcing user to create explicitly first? Right now the setter is internal, so it's not immediately clear to me.

Either way, this should either be `set(jwtSigner: ` or `setJWTSigner(_ signer: Signer` I lean towards the former